pkg/containertools/imagereader.go

```diff
@@ -53,7 +53,7 @@ func (b ImageLayerReader) GetImageData(image, outputDir string, opts ...GetImage
 		os.Mkdir(outputDir, 0777)
 	}
 
-	err := b.Cmd.Pull(image)
+	err := b.Cmd.Pull(image, options.SkipTLS)
```

Comment:
skipsTLS is an option for the bundle pulling but not for the index pulling? Do I understand that code right?

Reply:
nvm this PR is just about exposing that option to the tests